apps/accounts/tests/test_provider_request.py

```diff
@@ -50,7 +50,7 @@ def wizard_form_org_location_data():
     """
     return {
         "provider_registration_view-current_step": "1",
-        "locations__1-TOTAL_FORMS": "0",
+        "locations__1-TOTAL_FORMS": "1",
```

Comment:
this was needed - it looks like the formset checks look for this value, even if the actual number of forms submitted might have correct in the fixture